app/controllers/admin/topics_controller.rb

```diff
@@ -61,9 +61,6 @@ def destroy
       @topic.destroy
       flash[:success] = 'Topic deleted'
       redirect_to admin_topics_path
-    else 
-      flash[:danger] = 'You can not delete topic with questions!'
-      redirect_to admin_topics_path  
     end
```

Comment:
Coz we have no access to button delete when topic has questions. 
-if @questions.empty? 
        = link_to 'Delete topic', admin_topic_path(@topic), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger'
so i think we don't need this condition.